svc-hoppy-usage/src/docker/Dockerfile

```
@@ -0,0 +1,25 @@
+FROM python:3.9-alpine
```

To reduce SecRel maintenance, have it use the same base image as https://github.com/department-of-veterans-affairs/abd-vro/blob/1d804fe37a035fb607919e891acb5841e7677550/gradle-plugins/src/main/resources/docker/Dockerfile-python#L5.
```suggestion
FROM bitnami/python:3.10
```

---

Dockerfile was removed so this is moot. Tested the build with bitnami base image and it works great.